frontend-mobile/lib/storage/settings_storage.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+const String usernameKey = 'name';
+
+Future<String> getUsername() async {
```

Comment:
This would be a more functional approach. Iam not very familiar with the bloc lib, do you think this is a valid usecase?

Reply:
In general I'd avoid functions, because they are a pain when testing the application with mockito or mocktail.

Reply:
Alright, I will dive into tests next week once I have more time :) Will change it then